### ISSUE TITLE
False positive TS6133 (declared but never used) report.

### ISSUE THREAD

**Issue by Strate** [Microsoft/TypeScript#11985]
```
% tsc --version
Version 2.0.6
```

```ts
class SomeClass {
	private method = () => {

	}

	public handler() {
		const {method} = this;
		method();
	}
}
```
and then compile file: `tsc <file> --noUnusedLocals`

**Expected**
No errors

**Actual**
TS6133.ts(2,10): error TS6133: 'method' is declared but never used.

**Comment by wallverb** [comment]
Similar https://github.com/Microsoft/TypeScript/issues/11324 ?

**Comment by Strate** [comment]
@wallverb actually duplicate
